github-runner-manager/src/github_runner_manager/manager/runner_manager.py

```diff
@@ -354,6 +353,7 @@ def _delete_cloud_runners(
                 logger.error("No metrics returned after deleting %s", cloud_runner.instance_id)
             else:
                 extracted_runner_metrics.append(runner_metric)
+        CLEANED_RUNNERS_TOTAL.labels(self.manager_name).inc(len(cloud_runners))
```

Comment:
Couldn't we inc one by one in the loop (after line 328)? Otherwise we would again increase the amount also for non-deleted runners (if there is an exception).